.snyk

```diff
@@ -13,4 +13,15 @@ ignore:
     - node-pre-gyp > tar-pack > debug:
         reason: Not user facing
         expires: '2017-11-24T08:35:43.242Z'
-patch: {}
+  'npm:shelljs:20140723':
+    - eslint-config-nhsuk > eslint-plugin-json > jshint > shelljs:
+        reason: no patch available
+        expires: '2018-02-02T14:12:43.543Z'
+# patches apply the minimum changes required to fix a vulnerability
+patch:
```

Comment:
I didn't think patching actually did anything. Does the data on the ignores above (in the file) need updating and then the patching section can be removed?

Reply:
Deleting the `.snyk` file then re-running the wizard has removed the ignores